akka-docs/src/main/paradox/typed/cluster.md

```diff
@@ -247,7 +247,7 @@ If you don’t use RP, you should anyway carefully read the [documentation](http
 of the Split Brain Resolver and make sure that the solution you are using handles the concerns
 described there.
 
-### Auto-downing (DO NOT USE)
+### Auto-downing - DO NOT USE
```

Comment:
I think this is ok to change, based on what I'd changed on the other related PRs and looking at the git history, I don't think it breaks anything.